protractor.conf.js

```diff
@@ -5,7 +5,7 @@
 var SpecReporter = require('jasmine-spec-reporter').SpecReporter;
 
 exports.config = {
-  allScriptsTimeout: 11000,
+  allScriptsTimeout: 600000,
```

Comment:
I'd recommend removing all the changes to this file.  We only bumped up these two timeouts cause Travis was having issues communicating with our demo REST site.  Ideally, these timeouts (`allScriptsTimeout` and `defaultTimeoutInterval`) should both be low.  Since we are now running the REST in Docker, hopefully we can reset them.